standalone/content/js/webapp.jsx

```diff
@@ -72,6 +75,7 @@ loop.webapp = (function(_, OT, mozL10n) {
    */
   var PromoteFirefoxView = React.createClass({
     propTypes: {
+      activeRoomStore: React.PropTypes.instanceOf(loop.store.ActiveRoomStore).isRequired,
```

Comment:
I don't see the reason for adding this store and passing it down. Was it left in by accident?